spring-cloud-task-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -181,20 +187,21 @@ public class SampleTask {
 
 		@Override
 		public void run(String... strings) throws Exception {
-			System.out.println("Hello World!");
+			System.out.println("Hello, World!");
```

Comment:
So strict on a `Hello World` app.  We're taking over the world ;-)

Reply:
I'm firmly in the "details matter" camp - taking over the world one comma at a time! ;)